spec/popup/format-spec.js

```diff
@@ -0,0 +1,51 @@
+import format from '../../src/common/popup/utils/format.js';
+
+describe('format util', () => {
+  const ticket = {
+    id: 'BTC-042',
+    title: 'Weird Bug while eating cake',
+    type: 'bug'
+  };
+
+  describe('commit', () => {
+    it('formats', () => {
+      const formatted = format.commit(ticket);
+      expect(formatted).toBe(`[#${ticket.id}] ${ticket.title}`);
+    });
+  });
+
+  describe('branch', () => {
+    it('formats', () => {
```

Comment:
How about `it('includes ticket type, id and title', …)`? Might be a little more descriptive?